<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>573</width>
    <height>544</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="Mouse" name="btn0">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>90</y>
      <width>85</width>
      <height>80</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Mouse" name="btn1">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>90</y>
      <width>85</width>
      <height>80</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Mouse" name="btn2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>90</y>
      <width>85</width>
      <height>80</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Mouse" name="btn5">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>170</y>
      <width>85</width>
      <height>80</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Mouse" name="btn4">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>170</y>
      <width>85</width>
      <height>80</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Mouse" name="btn3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>170</y>
      <width>85</width>
      <height>80</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Mouse" name="btn8">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>250</y>
      <width>85</width>
      <height>80</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Mouse" name="btn7">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>250</y>
      <width>85</width>
      <height>80</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Mouse" name="btn6">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>250</y>
      <width>85</width>
      <height>80</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>20</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(20, 21, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
    <property name="value" stdset="0">
     <double>30.000000000000000</double>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>30</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(20, 21, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="digitCount">
     <number>3</number>
    </property>
    <property name="value" stdset="0">
     <double>0.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Baoli SC</family>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 32, 26);</string>
    </property>
    <property name="text">
     <string>时间：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>30</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Baoli SC</family>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 32, 26);</string>
    </property>
    <property name="text">
     <string>得分：</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>573</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>游戏</string>
    </property>
    <addaction name="actionStart"/>
    <addaction name="actionStop"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
    <addaction name="separator"/>
    <addaction name="actionReset"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionAboutGame"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>设置</string>
    </property>
    <widget class="QMenu" name="menu_4">
     <property name="title">
      <string>等级设置</string>
     </property>
     <addaction name="actionBaseMode"/>
     <addaction name="actionMidMode"/>
     <addaction name="actionHighMode"/>
     <addaction name="actionCrazyMode"/>
    </widget>
    <widget class="QMenu" name="menu_5">
     <property name="title">
      <string>背景音乐</string>
     </property>
     <addaction name="actionMusicOne"/>
     <addaction name="actionMusicTwo"/>
     <addaction name="actionMusicThree"/>
     <addaction name="actionMusicFour"/>
     <addaction name="actionMusicFive"/>
    </widget>
    <addaction name="menu_4"/>
    <addaction name="menu_5"/>
    <addaction name="actionMusicNext"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_3"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionStart"/>
   <addaction name="actionStop"/>
   <addaction name="separator"/>
   <addaction name="actionReset"/>
   <addaction name="actionExit"/>
   <addaction name="separator"/>
   <addaction name="actionAboutGame"/>
   <addaction name="actionAboutQt"/>
   <addaction name="separator"/>
   <addaction name="actionBaseMode"/>
   <addaction name="actionMidMode"/>
   <addaction name="actionHighMode"/>
   <addaction name="actionCrazyMode"/>
   <addaction name="separator"/>
   <addaction name="actionMusicNext"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionStart">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/start.png</normaloff>:/images/start.png</iconset>
   </property>
   <property name="text">
    <string>开始</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/stop.png</normaloff>:/images/stop.png</iconset>
   </property>
   <property name="text">
    <string>结束</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/nonstop.png</normaloff>:/images/nonstop.png</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/Refresh.png</normaloff>:/images/Refresh.png</iconset>
   </property>
   <property name="text">
    <string>重置</string>
   </property>
  </action>
  <action name="actionAboutGame">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/bt02.png</normaloff>:/images/bt02.png</iconset>
   </property>
   <property name="text">
    <string>关于游戏</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/qt1.png</normaloff>:/images/qt1.png</iconset>
   </property>
   <property name="text">
    <string>关于Qt</string>
   </property>
  </action>
  <action name="actionBaseMode">
   <property name="icon">
    <iconset>
     <normaloff>images/lvl01.png</normaloff>images/lvl01.png</iconset>
   </property>
   <property name="text">
    <string>初级模式</string>
   </property>
  </action>
  <action name="actionMidMode">
   <property name="icon">
    <iconset>
     <normaloff>images/lvl18.png</normaloff>images/lvl18.png</iconset>
   </property>
   <property name="text">
    <string>中级模式</string>
   </property>
  </action>
  <action name="actionHighMode">
   <property name="icon">
    <iconset>
     <normaloff>images/lvl21.png</normaloff>images/lvl21.png</iconset>
   </property>
   <property name="text">
    <string>高级模式</string>
   </property>
  </action>
  <action name="actionCrazyMode">
   <property name="icon">
    <iconset>
     <normaloff>images/lvl04.png</normaloff>images/lvl04.png</iconset>
   </property>
   <property name="text">
    <string>变态模式</string>
   </property>
  </action>
  <action name="actionMusicOne">
   <property name="text">
    <string>音乐1</string>
   </property>
  </action>
  <action name="actionMusicTwo">
   <property name="text">
    <string>音乐2</string>
   </property>
  </action>
  <action name="actionMusicThree">
   <property name="text">
    <string>音乐3</string>
   </property>
  </action>
  <action name="actionMusicFour">
   <property name="text">
    <string>音乐4</string>
   </property>
  </action>
  <action name="actionMusicFive">
   <property name="text">
    <string>音乐5</string>
   </property>
  </action>
  <action name="actionMusicNext">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/iTunes.icns</normaloff>:/images/iTunes.icns</iconset>
   </property>
   <property name="text">
    <string>下一首音乐</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Mouse</class>
   <extends>QPushButton</extends>
   <header>mouse.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
